packages/zosfiles/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe z/OS files SDK package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Added support for the `--encoding` flag to the `zowe upload dir-to-uss` to allow for encoding uploaded directories for command group consistency. [#2337](https://github.com/zowe/zowe-cli/issues/2337)
```

Comment:
Technically there is no need to have a changelog for the zosfiles sdk in this case.
It's fine to keep it, but if you plan to remove it, we may need to use the `no-changelog` label